http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -236,4 +271,11 @@ public static OidcClientConfiguration build(OidcJsonConfiguration oidcJsonConfig
         return new OidcClientConfigurationBuilder().internalBuild(oidcJsonConfiguration);
     }
 
+    private boolean isValidPath(String path) {
+        String tmpPath = path.trim();
+        if (tmpPath.length() > 1 && tmpPath.startsWith("/")) {
```

Comment:
Just a minor comment, this could be simplified to `return tmpPath.length() > 1 && tmpPath.startsWith("/")`

Reply:
done